test/Issues/RDBC_653.ts

```diff
@@ -0,0 +1,45 @@
+import {IDocumentStore} from "../../src";
+import { testContext, } from "../Utils/TestUtil";
+import {assertThat} from "../Utils/AssertExtensions";
+
+describe("RDBC-653", function () {
+
+    let store: IDocumentStore;
+
+    beforeEach(async function () {
```

Comment:
please add more test cases with falsy values in metadata and object.

Reply:
done